### ISSUE TITLE
Consider changes to UI Context Responses dependent on Time Control Node

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#319]
From https://github.com/phetsims/molecules-and-light/issues/299#issuecomment-604927266, we think this simulation would benefit many learners if the Pause/Play button was disabled when the light source is switched off.

We will consider updating or changing descriptions for this sim already outlined in #317 and #318 once Time Control Node has this option available.

This issue is also somewhat related to #295 and #296 

@jessegreenberg, I posted https://github.com/phetsims/scenery-phet/issues/592

**Comment by jessegreenberg** [comment]
@terracoda can you comment on what should be done with this issue now that https://github.com/phetsims/molecules-and-light/issues/320 and https://github.com/phetsims/scenery-phet/issues/592 are closed?

**Comment by terracoda** [comment]
@jessegreenberg, I think you can implement the context responses outline in #317 and #318 and close this issue.

This issue was just a place holder for discussion and a decision. The decision to not disable the the pause/play button happened in #320.

**Comment by terracoda** [comment]
I guess I can just close this issue actually :-)
